src/main/java/bio/terra/cda/app/model/NestedColumn.java

```diff
@@ -0,0 +1,58 @@
+package bio.terra.cda.app.model;
+
+/**
+ * Table cells that represent as nested structures must be unnested from it decendant heiarchy. the
+ * UNNEST clause is generated by unwinding the "dot" hierarchy such that:
+ * gdc-bg-sample.cda_mvp.v3.ResearchSubject.identifier.value would unwind to the following. "SELECT
+ * DISTINCT (_D.value) FROM gdc-bq-sample.cda_mvp.v3, UNNEST(ResearchSubject) as _C,
+ * UNNEST(_C.identifier) AS _D";
+ */
+public class NestedColumn {
+  private String column;
+  private String unnestClause;
```

Comment:
Since these fields are constant after the constructor completes, it would be better to declare them as `final` to make this clear. To do this, you could move the code in `generateUnnestClause()` to the constructor. Another approach is to change this constructor to accept `column` and `unnestClause` and replace `generateUnnestClause()` with a static factory method that accepts `columnName` and returns a new `NestedColumn`.